core/core-awt/pom.xml

```diff
@@ -62,6 +62,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
The root pom had this declaration set to 1.8.  I'm in the processes of getting the pom ready for the maven repository and will move to 11 as the default.  Is there any reason that this has to be set at the core-awt level instead of just the root?

Reply:
The project is not compilable with 1.8 I'm getting
C:\development\icepdf\core\core-awt\src\main\java\org\icepdf\core\pobjects\fonts\zfont\cmap\CMap.java:543:56
java: incompatible types: possible lossy conversion from int to char
C:\development\icepdf\core\core-awt\src\main\java\org\icepdf\core\pobjects\fonts\zfont\cmap\CMap.java:768:35
java: cannot find symbol
  symbol:   method readAllBytes()
  location: variable inputStream of type java.io.InputStream